core/assertions.ts

```diff
@@ -0,0 +1,39 @@
+import { isNonEmptyString, isUuidV4, UuidV4 } from './typeguards';
+
+export function assertNonEmptyString(
+  value: unknown,
+  msgOrExceptionFactory?: string | (() => Error) | Error,
+  /** auto-trim, default true */
+  trim?: boolean
+): asserts value is string {
+  const autoTrim = trim ?? true;
+  if (isNonEmptyString(value, autoTrim)) {
+    return;
+  }
+  if (typeof msgOrExceptionFactory === 'string' || msgOrExceptionFactory === undefined) {
```

Comment:
Codacy has a fix for the issue: [Replace `'string'` with `"string"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7711626)
```suggestion
  if (typeof msgOrExceptionFactory === "string" || msgOrExceptionFactory === undefined) {
```